src/Observers/ItemObserver.php

```diff
@@ -70,6 +70,12 @@ protected function cleanModel(MenuItem $item)
                 }
             }
 
+            if( request()->get('link_type') === MenuItem::linkTypesAttributes()[MenuItem::TYPE_EMPTY]) {
+                $item->is_empty_link = 1;
+            } else {
+                $item->is_empty_link = 0;
+            }
+
```

Comment:
I think you can remove this part because code above define the column with "null" and default value in database is "0"

Reply:
but on update if « is_empty_link » was 1, if i remove this part, « is_empty_link » can't be set to 0

Reply:
( TYPE_EMPTY not added in `linkTypesAttributes` method on purpose )